extensions/amp-story/0.1/media-pool.js

```diff
@@ -549,19 +536,21 @@ export class MediaPool {
    * a media element that can be used in its stead for playback.
    * @param {!HTMLMediaElement} domMediaEl The media element, found in the DOM,
    *     whose content should be loaded.
-   * @return {?HTMLMediaElement} A media element from the pool that can be used
+   * @return {Promise<!HTMLMediaElement>} A media element from the pool that can be used
    *     to replace the specified element.
    */
   loadInternal_(domMediaEl) {
     if (!isConnectedNode(domMediaEl)) {
       // Don't handle nodes that aren't even in the document.
-      return null;
+      return Promise.resolve();
```

Comment:
Should we reject the promise instead?

Reply:
I'm not sure this is really a rejection; we've done all the loading we want to do for that element, it just happens that we don't want to do any :)